packages/cli/test/integration/nodes.api.test.ts

```diff
@@ -64,10 +62,6 @@ beforeEach(async () => {
 	mocked(findInstalledPackage).mockReset();
 });
 
-afterAll(async () => {
-	await testDb.terminate();
```

Comment:
Is this handled by the clean up process of the test?

Reply:
This moved [here](https://github.com/n8n-io/n8n/pull/6654/files#diff-d9cac8044aada927b292ffc11b0cd962d808f27c08f220c91e552ce0a5576cddR270), so we don't need to keep repeating this in every test.